robot_ws/src/person_detection_robot/src/person_detection_robot/rekognize_tts.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+"""
+ Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy of this
+ software and associated documentation files (the "Software"), to deal in the Software
+ without restriction, including without limitation the rights to use, copy, modify,
+ merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+ permit persons to whom the Software is furnished to do so.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+
+import os
+import time
+import json
+
+import gi
+gi.require_version('Gst', '1.0')
+from gi.repository import Gst
+
+import rclpy
+from rclpy.node import Node
+from rclpy.time import Time
+from rclpy.action import ActionClient
+
+from tts_interfaces.srv import Synthesizer
+from std_msgs.msg import String
+
+class Rekognizer(Node):
+
```

Comment:
I'm not a fan of this approach of copying basically all of [voicer.tts](https://github.com/aws-robotics/tts-ros2/blob/master/tts/tts/voicer.py), it'll be difficult to maintain and troubleshoot.
Instead, can you import the `Voicer` node from tts/voicer.py and use it here? Shouldn't be too difficult, if I'm not mistaken you'll just need to modify the [`speak`](https://github.com/aws-robotics/tts-ros2/blob/master/tts/tts/voicer.py#L75) method to accept the text and metadata parameters instead of directly accessing sys.argv.

Reply:
> Since `raspicam` isn't working on ROS2 Dashing, how was this sample app tested? Because my understanding is testing locally on a PC will result in failing to open `/dev/ttyACM0`.

There are 2 boards on Turtlebot3
1. Raspberry Pi 3B+ (To Run ROS related application)
2. STM OpenCR Board (For Wheel Related Operations and not for raspicam)

"/dev/ttyACM0" is for STM board and not for raspicam. Please find details ([tb3 page](http://emanual.robotis.com/docs/en/platform/turtlebot3/ros2/#upload-ros-2-firmware-of-turtlebot3-to-opencr))
Turtlebot3 bring up related operation, You can not run on PC. You need to run on turtlebot3 hardware.
If you need to run on PC, comment on [turtlebot3_bringup_launch](https://github.com/sandiprakhasiya/aws-robomaker-sample-application-persondetection/blob/6be135bfd951c9e71e97f43b407e246562484d81/robot_ws/src/person_detection_robot/launch/deploy_person_detection.launch.py#L92) in deploy_person_detection.launch.py file (turtlebot3 HW related operations).

And raspicam/usbcam node does not initiate directly (it is conditional). 
If Run with USB Camera
ros2 launch person_detection_robot deploy_person_detection.launch.py run_usb_cam:True
If Run with PI Camera (Currently Not Working with Raspicam)
ros2 launch person_detection_robot deploy_person_detection.launch.py run_pi_cam:True